### PR TITLE
Rubyists Guide to Ember Dependencies

### PR DESCRIPTION
**Pull request by michaeldupuisjr**
Adds a table and fleshes stuff out a bit more.

### CODE REVIEW

File: source/stylesheets/base.sass [hunk 1]
```diff
@@ -69,3 +69,17 @@ a
     family: "icomoon"
   content: attr(data-icon)
   speak: none
+
```
Comment (bcardarella) [inline source/stylesheets/base.sass]: you should have @acacheung sign off on these styles

File: source/posts/2015-02-12-rubyists-guide-to-ember-dependencies.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+One of the early hurdles a Ruby developer faces when working on an Ember.js application is dependency management. A popular mechanism for managing a Ruby application’s dependencies is the [Gemfile](http://bundler.io/gemfile.html) provided by [Bundler](http://bundler.io/ ). Including a library is as easy as declaring it in the Gemfile and running `bundle install`:
+
+```ruby
+# Gemfile
+source 'https://rubygems.org'
+gem 'rails', '~> 4.2.0'
+```
+
+Life isn’t so easy in Ember land. For better or worse, there is no dominant, single package manager in JavaScript. Ember applications, and more specifically, those running [Ember-CLI](http://www.ember-cli.com/), rely on two package managers: [Bower](http://bower.io/) for client-side libraries and [NPM](https://www.npmjs.com/) for server-side libraries.
```
Comment (bcardarella) [inline source/posts/2015-02-12-rubyists-guide-to-ember-dependencies.md]: I would remove this first sentence, it kind of reads as if Ember is to blame for the state of JS package management

File: source/posts/2015-02-12-rubyists-guide-to-ember-dependencies.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+In this post, I'll provide a basic dependency management primer for
+those moving from Ruby to JavaScript.
+
+# NPM
```
Comment (bcardarella) [inline source/posts/2015-02-12-rubyists-guide-to-ember-dependencies.md]: I *think* `npm` is always lowercased as evidenced by all of the lowercase `npm`s on https://www.npmjs.com/

File: source/posts/2015-02-12-rubyists-guide-to-ember-dependencies.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+Ember-CLI uses NPM to manage internal dependencies. NPM resembles Rubygems, in so far as it allows you to install and manage third-party libraries, which in this case, are Node.js programs.
```
Comment (bcardarella) [inline source/posts/2015-02-12-rubyists-guide-to-ember-dependencies.md]: the `G` is capitalized in `RubyGems`

File: source/posts/2015-02-12-rubyists-guide-to-ember-dependencies.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+## package.json
+Libraries for NPM are referred to as “packages.” Each package has a `package.json` file which lists the dependencies of the library itself. In this regard, the `package.json` is analogous to a Ruby gem’s `gemspec` file.
```
Comment (bcardarella) [inline source/posts/2015-02-12-rubyists-guide-to-ember-dependencies.md]: "Ruby gem's" => "RubyGem's"

File: source/posts/2015-02-12-rubyists-guide-to-ember-dependencies.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+package managers we discussed:
+
+| Description  | Ruby | JS (NPM, server-side) | JS (Bower, client-side) |
+| ---- | ---- | ---- | ---- |
+| Term for external library | "Gem" | "Package" | "Component" |
+| End-user configuration file | `.gemrc` | `.npmrc` | `.bowerrc` |
+| Per-library configuration file | `*.gemspec` | `package.json` | `bower.json` |
+| Cache directory | `~/.gem/` | `~/.npm/` | `~/.cache/bower/` |
+
+As ES6 becomes more prevalent and JavaScript code becomes more
```
Comment (bcardarella) [inline source/posts/2015-02-12-rubyists-guide-to-ember-dependencies.md]: ES6 has been renamed as ES2015 but I don't know if many people know this

**Comment by michaeldupuisjr** [comment]
Date has been updated.

**Comment by danmcclain** [comment]
Update date again please, and we'll merge as soon as you do. Thanks

**Comment by michaeldupuisjr** [comment]
@danmcclain Updated.

**Comment by danmcclain** [comment]
Finally published!

**Pull request merged**